source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1314,6 +1315,15 @@ message TransformationToggleAccessChainInstruction {
 
 }
 
+message TransformationToggleNullConstant {
+
+  // A transformation that replaces a null boolean, int or floating-point null constant (OpConstantNull) with
+  // a zero-valued constant of the same type (OpConstant), and vice versa
+
+  // The result id of the constant declaration instruction
+  uint32 constant_id = 1;
```

Comment:
Again, this is sort of obsolete given my overall comment on the PR, but is perhaps worth digesting anyway.

I think it would be better for us to do this on a per-use basis.

For example, if %a is OpConstantFalse of type bool, and %b is OpConstantNull of type bool, and if the module contains many uses of %a then it should be possible to transform just one of those uses to %b.

To do this, use an IdUseDescriptor.